src/tools/rados/rados.cc

```diff
@@ -238,6 +244,173 @@ void usage(ostream& out)
 "    --omap-key-file file            read the omap key from a file\n";
 }
 
+// Handle conditional libradosstriper support:
+enum class UseStriper { yes, no };
```

Comment:
why not `bool` ?

Reply:
I went back and forth on this one. In this particular case, I could go either way, but because read() already has plenty of parameters, I like being able to see what that flag's for at any call site rather than having to go look.

Reply:
i think that's what the parameter *name* is for.